.gitattributes

```diff
@@ -12,8 +12,7 @@
 *.step filter=lfs diff=lfs merge=lfs -text
 *.FCStd* filter=lfs diff=lfs merge=lfs -text
 
-# Schematics (KiCad)
+# Electronics Design (KiCad)
 *.kicad_pcb filter=lfs diff=lfs merge=lfs -text
-*.pro filter=lfs diff=lfs merge=lfs -text
-*.lib filter=lfs diff=lfs merge=lfs -text
-*.dcm filter=lfs diff=lfs merge=lfs -text
+*.sch filter=lfs diff=lfs merge=lfs -text
```

Comment:
The schematics are stored in a quite readable S-expression format, and glancing over the diff to see added components and changed fields has been quite helpful in the past. They aren't overly large either, so I would vouch against using LFS to track them.

Reply:
Fair enough. I'll untrack schematics from git LFS.